### ISSUE TITLE
❓ Question: v0.18 - EACCES: permission denied

### ISSUE THREAD

**Issue by BeardedWatermelon** [stonith404/pingvin-share#264]
### 🙋‍♂️ Question

Running v0.18 in Docker on Mac. Application seems to work just fine, didn't have to change permissions as per the release notes but pulled the image again and recreated the container like a new installation just to be safe. The question is about the following log entry that I can see in the logs. 

```
2023-09-21 21:42:27 Failed to write image to cache pTG8HzA1UyZcKw5Z2ovMKRzp3VhidC+kBdsO7cBfExE= [Error: EACCES: permission denied, mkdir '/opt/app/frontend/.next/cache'] {
2023-09-21 21:42:27   errno: -13,
2023-09-21 21:42:27   code: 'EACCES',
2023-09-21 21:42:27   syscall: 'mkdir',
2023-09-21 21:42:27   path: '/opt/app/frontend/.next/cache'
2023-09-21 21:42:27 }
```

In the docker terminal from inside the container on Docker Desktop for Mac, I ran sudo chown -R 1000:1000 data but i get a message "sudo not found". If i don't use sudo, I get "data: no such file or directory".

whoami returns "node" so obviously the container runs under the node user and not root. 

ls -l /opt/app/backend returns: 
```
drwxr-xr-x    9 node     node           288 Sep 21 18:43 data
drwxr-xr-x    4 root     root          4096 Sep 21 14:40 dist
drwxr-xr-x  271 root     root         12288 Sep 21 14:41 node_modules
-rw-r--r--    1 root     root          2541 Sep 21 14:31 package.json
drwxr-xr-x    4 root     root          4096 Sep 21 14:31 prisma
```
ls -l /opt/app/frontend returns:
```
drwxr-xr-x  141 root     root          4096 Sep 21 14:49 node_modules
-rw-r--r--    1 root     root          1433 Sep 21 14:49 package.json
drwxr-xr-x    3 root     root          4096 Sep 21 14:45 public
-rw-r--r--    1 root     root          4739 Sep 21 14:49 server.js
drwxr-xr-x    3 root     root          4096 Sep 21 14:49 src
```

and ls -l /opt/app/frontend/.next returns:
```
-rw-r--r--    1 root     root            21 Sep 21 14:49 BUILD_ID
-rw-r--r--    1 root     root          7938 Sep 21 14:49 build-manifest.json
-rw-r--r--    1 root     root            20 Sep 21 14:48 package.json
-rw-r--r--    1 root     root           312 Sep 21 14:49 prerender-manifest.json
-rw-r--r--    1 root     root             2 Sep 21 14:49 react-loadable-manifest.json
-rw-r--r--    1 root     root          3551 Sep 21 14:49 required-server-files.json
-rw-r--r--    1 root     root          3565 Sep 21 14:49 routes-manifest.json
drwxr-xr-x    5 root     root          4096 Sep 21 14:49 server
drwxr-xr-x    4 root     root          4096 Sep 21 14:45 static
```

chown -R node /opt/app/frontend returns: 
```
chown: /opt/app/frontend/.next/package.json: Operation not permitted
chown: /opt/app/frontend/.next/routes-manifest.json: Operation not permitted
........ -----------------------(it basically returns Operation not permitted for every file under frontend)
chown: /opt/app/frontend/public: Operation not permitted
chown: /opt/app/frontend/public: Operation not permitted
```

mkdir /opt/app/frontend/.next/cache returns: 
`mkdir: can't create directory '/opt/app/frontend/.next/cache': Permission denied
`
and sudo mkdir /opt/app/frontend/.next/cache returns:
`/bin/sh: sudo: not found`

Just to clarify, I'm using docker desktop on Mac, selecting the running container, then the Terminal Tab. As far as I know, due to MacOS Limitations, Docker for Mac creates a linux virtual machine where the containers are ran. 

Again, the application seems to work fine. Should I be worried about this log entry? Or should I ignore it?

**Comment by stonith404** [comment]
Should be fixed in `v0.18.1` can you try it?

**Comment by stonith404** [comment]
Yeah, I changed the user back to `root` because it has broken many existing installations, so this is expected. You don't have to change permissions anymore, including the `chmod? command.

By the way, thank you for providing so much information like you did above. I really appreciate that because it helps me find and solve problems faster :)
